.github/workflows/examples_js.yml

```diff
@@ -0,0 +1,92 @@
+name: JS Example Tests
+
+on:
+  workflow_call:
+    inputs:
+      working-directory:
+        required: true
+        type: string
+      api_key:
+        required: false
+        type: string
+      base_url:
+        required: false
+        type: string
+      author:
+        required: false
+        type: string
+      commit_message:
+        required: false
+        type: string
+      dont_notify:
+        required: false
+        type: boolean
+  workflow_dispatch:
+    inputs:
+      working-directory:
+        required: true
+        type: string
```

Comment:
Why is it required? Same for author and commit_message.

We can ask for triggered_by string which can be used with external repo, branch, commit, etc.

For eg:- we can trigger this workflow from hermes.

Reply:
For Author and Commit_Message it's false

Reply:
Can have one input triggered_by. Possible values 
-`https://github.com/ComposioHQ/hermes/commit/93e20627ed14277f9f48adaa74d9d73d913e9378`
- https://github.com/ComposioHQ/composio
- manual_check

Otherwise in case of push or pull this will always be available.

With that it will easier to reference also. WDYT? your call here.

Reply:
Makes sense, changing this to false for now